uikit/src/main/java/com/midtrans/sdk/uikit/views/shopeepay/payment/ShopeePayPaymentActivity.java

```diff
@@ -156,20 +137,14 @@ public void onPaymentError(Throwable error) {
     private void initPresenter() {
         presenter = new ShopeePayPaymentPresenter(this);
         presenter.setTabletDevice(this);
-        presenter.setShopeeInstalled(this);
     }
 
     private void initLayout() {
         ViewStub stub = findViewById(R.id.shopee_layout_stub);
         if (isTablet()) {
             stub.setLayoutResource(R.layout.uikit_layout_shopeepay_payment_tablet);
         } else {
-            if (isProductionBuild()) {
-                stub.setLayoutResource(isShopeeInstalled() ? R.layout.uikit_layout_shopeepay_payment
-                    : R.layout.layout_install_shopeepay);
```

Comment:
is this layout still needed?

Reply:
this one is for the payment instruction on deeplink right? if so, then it still going to be used.

Reply:
sorry i mean this one `R.layout.layout_install_shopeepay` since it's only called by the removed codes.

Reply:
owh you mean the XML file for that layout right???

Reply:
yes kang

Reply:
it was removed...

Reply:
cool!